cypress/integration/wheel.spec.ts

```diff
@@ -0,0 +1,40 @@
+describe("cy.realMouseWheel", () => {
+
+  it("scroll to find a day", () => {
+    cy.visit("https://anvil.servicetitan.com/components/date-picker/")
```

Comment:
Please move the code to our fixtures (preferred) or at least to code sandbox so we will own it.

Reply:
moved to use a fixture file.